src/main/java/frc/robot/subsystems/ChassisSubsystem.java

```diff
@@ -36,8 +36,8 @@
   private RelativeEncoder rearLeftEncoder_ = null;
   private RelativeEncoder rearRightEncoder_ = null;
 
-  private MotorControllerGroup leftMotors_ = new MotorControllerGroup(frontLeftMotor_, rearLeftMotor_);
-  private MotorControllerGroup rightMotors_ = new MotorControllerGroup(frontRightMotor_, rearRightMotor_);
+  private MotorControllerGroup leftMotors_ = new MotorControllerGroup(frontLeftMotor_);
```

Comment:
If you are only using one motorController, you don't need the MotorControllerGroup calls. Just leave them out.